inlong-audit/audit-proxy/src/main/java/org/apache/inlong/audit/source/SimpleTcpSource.java

```diff
@@ -181,7 +181,7 @@ public synchronized void start() {
             }
         } catch (Exception e) {
             logger.error("Simple TCP Source error bind host {} port {},program will exit!", host,
```

Comment:
There is something wrong with the log description, could you please modify it to:
```suggestion
            logger.error("Simple TCP Source bind {}:{} error, the program will exit!", host, port, e);
```